Form-Controls/index.html

```diff
@@ -6,22 +6,112 @@
+        border: 1px solid #ccc;
+        border-radius: 5px;
+        font-size: 14px;
+      }
+      input:focus, select:focus {
+        border-color: #007bff;
+        outline: none;
+      }
+      button {
+        background-color: #007bff;
+        color: white;
+        font-weight: bold;
+        cursor: pointer;
+        transition: background-color 0.3s;
+      }
+      button:hover {
+        background-color: #0056b3;
+      }
+      .note {
+        font-size: 12px;
+        color: #999;
+      }
+    </style>
+  
   </head>
-  <body>
-    <header>
-      <h1>Product Pick</h1>
-    </header>
-    <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+<body>
+  <div class="big-room">
+    <h2>Customer Information</h2>
+    <form action="submit-form.php" method="POST">
+      <!--What is the customer's name? I must collect this data, and validate it. But what is a valid name? I must decide something.
+-->
+      <div class="form-group">
+        <label for="name">Customer Name:</label>
+        <input type="text" id="name" name="name" required pattern="^[A-Za-z\s]{2,50}$" 
+               title="Name must be 2-50 characters and contain only letters and spaces.">
+        <div class="note">Only letters and spaces are allowed (2-50 characters).</div>
+      </div>
+      
+      <!-- What is the customer's email? I must make sure the email is valid. Email addresses have a consistent pattern.
+ -->
+      <div class="form-group">
+        <label for="email">Email Address:</label>
+        <input type="email" id="email" name="email" required 
+               title="Please enter a valid email address.">
+      </div>
+      
+      <!-- What colour should this t-shirt be? I must give 3 options. How will I make sure they don't pick other colours?
+ -->
+      <div class="form-group">
+        <label for="color">T-Shirt Color:</label>
+        <select id="color" name="color" required>
+          <option value="red">Red</option>
+          <option value="blue">Blue</option>
+          <option value="green">Green</option>
+        </select>
+      </div>
+      
+      <!--What size does the customer want? I must give the following 6 options: XS, S, M, L, XL, XXL-->
+      <div class="form-group">
+        <label for="size">T-Shirt Size:</label>
+        <select id="size" name="size" required>
+          <option value="XS">XS</option>
+          <option value="S">S</option>
+          <option value="M">M</option>
+          <option value="L">L</option>
+          <option value="XL">XL</option>
+          <option value="XXL">XXL</option>
+        </select>
+      </div>
+      
+      <button type="submit">Submit</button>
       </form>
     </main>
     <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
+      <h2>By Mikiyas Gebremichael</h2>
```

Comment:
I don't think that heading level 2 is appropriate in this context. Please, think of other tag here.